src/utils/validation.js

```diff
@@ -72,6 +128,15 @@ const handleCourseEditFail = (errors) => {
   }
 };
 
+const handleCourseCreateFail = (errors) => {
+  const fieldName = getFieldName(errors);
+  if (fieldName) {
+    const node = document.getElementsByName(fieldName)[0];
+
+    if (node) { node.scrollIntoView({ behavior: 'smooth' }); }
+  }
+};
+
```

Comment:
should this live in CourseCreateForm?

Reply:
`handleCourseEditFail` also lives in the same file so I though it would be better to follow the precedent